workspace.html

```diff
@@ -40,52 +40,52 @@ <h4 class="heading-subtitle"><i>Get your best work done here.</i></h4>
             <h3>See the Space</h3>
             
           </div>
-          <div class="col-xs-6 col-md-4">
-            <div class="lgx-video-area">
-              <div class="video-thumbnail">
-                <a class="icon" data-toggle="modal" data-target="#see-space-video-modal" tab-index="-1">
-                  <img class="opaque" src="/images/banners/awesome-inc-space-banner.jpg">
-                  <i class="fa fa-play " aria-hidden="true"></i>
-                </a>
-              </div>  
-              <img src="/images/icons/arrow.png" id="video-arrow">
-              {% include modal-form.html initiative='tour' style='img-button' %}
-              <!-- Video Modal-->
-              <div id="see-space-video-modal" class="modal fade lgx-modal">
-                <div class="modal-dialog">
-                  <div class="modal-content">
-                    <div class="modal-header">
-                      <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    </div>
-                    <div class="modal-body" data-embed="0YgGYadr7rk">
-                      <iframe id="modalvideo" src="https://www.youtube.com/embed/0YgGYadr7rk" allowfullscreen></iframe>
-                    </div>
-                </div>
-              </div> <!-- //.Modal-->
-            </div><!--//.VIDEO-AREA-->
-            
-          </div><!--//.COL-->
-        </div><!--//.ROW-->
-      </div><!-- //.CONTAINER -->
-    </div><!-- //.INNER -->
-  </div><!-- //.SPOTLIGHT --> 
-</section>
+ <div class="col-xs-6 col-md-4">
+   <div class="lgx-video-area">
+     <div class="video-thumbnail">
+       <a class="icon" data-toggle="modal" data-target="#see-space-video-modal" tab-index="-1">
+         <img class="opaque" src="/images/banners/awesome-inc-space-banner.jpg">
+         <i class="fa fa-play " aria-hidden="true"></i>
+       </a>
+     </div>
+         <img src="/images/icons/arrow.png" id="video-arrow">
+         {% include modal-form.html initiative='tour' style='img-button' %}
+         <!-- Video Modal-->
+         <div id="see-space-video-modal" class="lgx-modal modal fade">
+           <body>
+             <div class="container">
+               <!-- Modal -->
+                   <!-- Modal content-->
+                   <div class="modal-content">
+                     <div class="modal-header">
+                       <button type="button" class="close" data-dismiss="modal">X</button>
+                     </div>
+                     <div class="modal-body">
+                       <iframe width="1110" height="450" src="https://www.youtube.com/embed/0YgGYadr7rk" frameboarder="0" allowfullscreen></iframe>
```

Comment:
@sgha229 Can we make this responsive instead of declaring the width in the iframe?